mopidy/mpd/translator.py

```diff
@@ -90,17 +101,23 @@ def track_to_mpd_format(track, position=None, stream_title=None):
     return result
 
 
-def artists_to_mpd_format(artists):
+def concatenate_multiple_values(artists, attribute):
     """
-    Format track artists for output to MPD client.
+    Format track artist values for output to MPD client.
```

Comment:
Is this function useful for other things than artists? There's nothing artist specific here any longer.

Maybe just replace all artist references here with "model" right away?